docs/ddl-of-carbondata.md

```diff
@@ -811,6 +811,7 @@ Users can specify which columns to include and exclude for local dictionary gene
        new SORT_COLUMNS.  
        
        UNSET is not supported, but it can set SORT_COLUMNS to empty string instead of using UNSET.
+       NOTE: SORT_SCOPE should not be NO_SORT, when set SORT_COLUMNS to empty
```

Comment:
when `SORT_COLUMNS` is empty, there is no significance to `SORT_SCOPE`. By default, it becomes `NO_SORT` always. So if we need to add validation we should not allow `SORT_SCOPE` at all in case of empty SORT_COLUMNS

Reply:
This scenario, first user creates a table with sort_scope='local_sort' and then set sort_columns=empty, using alter set tableproperties, which is not allowed